guides/common/modules/proc_uploading-additional-scap-content.adoc

```diff
@@ -0,0 +1,36 @@
+[id="Uploading_Additional_SCAP_Content_{context}"]
+= Uploading Additional SCAP Content
+
+You can upload additional SCAP content into {ProjectServer}, either content created by yourself or obtained elsewhere.
+For example, you can get the latest SCAP contents for additional systems from the https://github.com/ComplianceAsCode/content/releases[SSG GitHub repository].
+To use the CLI instead of the {ProjectWebUI}, see the xref:CLI_Uploading_Extra_SCAP_Content_{context}[CLI procedure].
+
+.Prerequisite
+* Your user account has the `create_scap_contents` permission.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *SCAP contents*.
+. Click *Upload New SCAP Content*.
+. Enter a title in the *Title* text box, such as `_My SCAP Content_`.
+. In *Scap File*, click *Choose file*, navigate to the location containing the SCAP data stream file and click *Open*.
+. On the *Locations* tab, select locations.
+. On the *Organizations* tab, select organizations.
+. Click *Submit*.
+
+If the SCAP content file is loaded successfully, a message similar to `Successfully created _My SCAP Content_` is displayed.
+
+[id="CLI_Uploading_Extra_SCAP_Content_{context}"]
+.CLI procedure
+* Run the following Hammer command on {ProjectServer}:
++
+[options="nowrap", subs="+quotes,attributes,verbatim"]
+----
+# hammer scap-content bulk-upload --type directory \
+--directory _/usr/share/xml/scap/my_content/_ \
```

Comment:
Is that directory required? Can users upload from "/media/my_disk/" or similar as well?

If it's a requirements, please note it above. If not, keep as is.

Reply:
It can probably be any directory.